juriscraper/opinions/united_states/state/texbizct.py

```diff
@@ -0,0 +1,122 @@
+# Scraper for Texas Business Court
+# CourtID: texb
+# Court Short Name: TX
+# Court Contacts:
+#  - http://www.txcourts.gov/contact-us/
+#  - Blake Hawthorne <Blake.Hawthorne@txcourts.gov>
+#  - Eddie Murillo <Eddie.Murillo@txcourts.gov>
+#  - Judicial Info <JudInfo@txcourts.gov>
+# Author: Luis Manzur
+# History:
+#  - 2025-07-29: Created by Luis Manzur
+import re
+from datetime import datetime
+
+from juriscraper.lib.log_tools import make_default_logger
+from juriscraper.lib.string_utils import titlecase
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+logger = make_default_logger()
+
+
+class Site(OpinionSiteLinear):
+    case_info_pattern = r"^(?P<docket>\S+)\s+(?P<name>.+?)(?:\s+(?P<citation>\d{4} Tex\. Bus\. \d+))?(?:\s+(?:Opinion|Memorandum|Denied|Some Defendants Dismissed|Remand|Reconsideration|Me|MSJ|PTJ|Rule).*)?$"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.url = "https://www.txcourts.gov/businesscourt/opinions/"
+        self.court_id = self.__module__
+        self.should_have_results = True
+
+    def _process_html(self) -> None:
+        """Parses the HTML content
+
+        :return None
+        """
+
+        links = self.html.xpath('//div[@class="panel-content"]//a')
+        for link in links:
+            text = link.get("title")
+            short_title = link.text_content()
+            if not text:
+                logger.debug("Skipping link with empty text: %s", link)
+                continue
+
+            summary = ""
+            summary_list = link.xpath("following::li[1]/text()")
+            if summary_list:
+                summary = summary_list[0].strip()
+
+            match = re.match(self.case_info_pattern, text)
+
+            split_short_title = short_title.split(",")
+            name = (
+                titlecase(match.group("name").strip())
+                if match
+                else split_short_title[0].strip()
+            )
+            citation = (
+                split_short_title[-1].strip()
+                if short_title
+                else (match.group("citation") or "")
+            )
+
+            date = (
+                f"{citation[:4]}/07/01"
+                if citation
+                else f"{datetime.now().year}/07/01"
+            )
```

Comment:
I dont like this.  this is going to create future dates, about half of the time.  And im not sure that will be allowed on CL.  instead I have an idea for catching our date approximates.